README.md

```diff
@@ -62,9 +50,9 @@ Person Name | [TextPersonNameAnnotation](https://github.com/nlpsandbox/nlpsandbo
 
 ## Specification
 
+- This NLP Sandbox tool version: TBA
 - NLP Sandbox schemas version: 1.2.0
-- NLP Sandbox tool version: 1.2.1
-- Docker image: [docker.synapse.org/syn22277123/phi-annotator-example]
+- Docker image: [docker.synapse.org/syn22277123/phi-annotator-huggingface-bert]
```

Comment:
This link appears to be dead?

Reply:
@cascadianblue Nothing we can do about it before we actually push the image(s).